src/routes/RequestsRoute.js

```diff
@@ -190,6 +229,17 @@ export const getListFormatter = (getRowURL, setURL) => ({
       <FormattedTime value={rq.requestDate} day="numeric" month="numeric" year="numeric" />
     </AppIcon>
   ),
+  'singlePrint': rq => (
+    <>
+      <SinglePrintButtonForPickSlip
+        rq={rq}
+        pickSlipsToCheck={pickSlipsToCheck}
+        pickSlipsPrintTemplate={pickSlipsPrintTemplate}
+        onBeforeGetContentForSinglePrintButton={onBeforeGetContentForSinglePrintButton}
+        pickSlipsData={pickSlipsData}
+        getPrintContentRef={getPrintContentRef}
+      />
+    </>),
```

Comment:
It is not necessary to have additional wrapper
```suggestion
      <SinglePrintButtonForPickSlip
        rq={rq}
        pickSlipsToCheck={pickSlipsToCheck}
        pickSlipsPrintTemplate={pickSlipsPrintTemplate}
        onBeforeGetContentForSinglePrintButton={onBeforeGetContentForSinglePrintButton}
        pickSlipsData={pickSlipsData}
        getPrintContentRef={getPrintContentRef}
      />),
```